<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>800</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget{
background-color: qlineargradient(spread:pad, x1:0.064, y1:0.107727, x2:0.869, y2:0.79, stop:0 rgba(23, 26, 33, 255), stop:0.806147 rgba(42, 71, 94, 255));}</string>
    </property>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1200</width>
       <height>800</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 170, 170);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="image">
      <property name="geometry">
       <rect>
        <x>344</x>
        <y>144</y>
        <width>512</width>
        <height>512</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="imageName">
      <property name="geometry">
       <rect>
        <x>545</x>
        <y>80</y>
        <width>110</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(199, 213, 224);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="index">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>686</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(199, 213, 224);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QSlider" name="verticalSlider">
      <property name="geometry">
       <rect>
        <x>1023</x>
        <y>144</y>
        <width>20</width>
        <height>512</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="invertedAppearance">
       <bool>true</bool>
      </property>
      <property name="tickPosition">
       <enum>QSlider::NoTicks</enum>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="loading">
     <property name="geometry">
      <rect>
       <x>536</x>
       <y>336</y>
       <width>128</width>
       <height>128</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="loadingLabel">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>260</y>
       <width>460</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>36</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(199, 213, 224);
font: 36pt &quot;Verdana&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionGoBack"/>
   <addaction name="separator"/>
   <addaction name="actionLeft_Slice"/>
   <addaction name="actionNext_Slice"/>
   <addaction name="actionReset"/>
  </widget>
  <action name="actionNext_Slice">
   <property name="icon">
    <iconset>
     <normaloff>../assets/icons/bonus/icons-32/arrow.png</normaloff>../assets/icons/bonus/icons-32/arrow.png</iconset>
   </property>
   <property name="text">
    <string>Next Slice</string>
   </property>
   <property name="toolTip">
    <string>Slide to the next slice</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionLeft_Slice">
   <property name="icon">
    <iconset>
     <normaloff>../assets/icons/bonus/icons-32/arrow-180.png</normaloff>../assets/icons/bonus/icons-32/arrow-180.png</iconset>
   </property>
   <property name="text">
    <string>Left Slice</string>
   </property>
   <property name="toolTip">
    <string>Slide to the next slice</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset>
     <normaloff>../assets/icons/bonus/icons-24/magnifier-zoom-in.png</normaloff>../assets/icons/bonus/icons-24/magnifier-zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="toolTip">
    <string>Zoom in the slice</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="icon">
    <iconset>
     <normaloff>../assets/icons/bonus/icons-24/magnifier-zoom-out.png</normaloff>../assets/icons/bonus/icons-24/magnifier-zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="toolTip">
    <string>Zoom out the slice</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="actionReset_Zoom">
   <property name="icon">
    <iconset>
     <normaloff>../assets/icons/bonus/icons-24/magnifier-zoom-fit.png</normaloff>../assets/icons/bonus/icons-24/magnifier-zoom-fit.png</iconset>
   </property>
   <property name="text">
    <string>Reset Zoom</string>
   </property>
   <property name="toolTip">
    <string>Reset to the normal scale</string>
   </property>
   <property name="shortcut">
    <string>0</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset>
     <normaloff>../assets/icons/icons/control.png</normaloff>../assets/icons/icons/control.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="toolTip">
    <string>Iterate through slices automaticly</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionpause">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../assets/icons/icons/control-pause.png</normaloff>../assets/icons/icons/control-pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="toolTip">
    <string>Pause</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset>
     <normaloff>../assets/icons/bonus/icons-24/arrow-circle-225.png</normaloff>../assets/icons/bonus/icons-24/arrow-circle-225.png</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="toolTip">
    <string>Reset</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionGoBack">
   <property name="icon">
    <iconset>
     <normaloff>../assets/icons/bonus/icons-24/arrow-curve-180.png</normaloff>../assets/icons/bonus/icons-24/arrow-curve-180.png</iconset>
   </property>
   <property name="text">
    <string>GoBack</string>
   </property>
   <property name="toolTip">
    <string>Go back to the upload Screen</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
